pkg/generator/main.go

```diff
@@ -140,6 +140,11 @@ func channelTweaks(channel string, props map[string]apiext.JSONSchemaProps) map[
 			continue
 		}
 
+		if channel == "standard" && name == "egress" {
```

Comment:
logically speaking this is somewhat hardcoded atm and we should do something similar to https://github.com/kubernetes-sigs/gateway-api/blob/be4b4051415179e2f207a13e3b78c07610fd7a37/pkg/generator/main.go#L172 but I didn't find the need yet to generalize standard and experimental validations separately just for this one use case, we can do it in the future if we have more such circumstances.

Reply:
yeah @astoycos you were right ptal ^ I think we are good now.

Reply:
++ I think this makes sense